cmd/build/helmify/kustomization.yaml

```diff
@@ -32,3 +32,30 @@ patchesJson6902:
       kind: CustomResourceDefinition
       name: configs.config.gatekeeper.sh
     path: labels_patch.yaml
+  # these are defined in the chart values rather than hard-coded
+  - target:
+      kind: Deployment
+      name: gatekeeper-audit
```

Comment:
Getting the following error:
```
kustomize build cmd/build/helmify | go run cmd/build/helmify/*.go
Error: no matches for OriginalId ~G_~V_Deployment|~X|gatekeeper-audit; no matches for CurrentId ~G_~V_Deployment|~X|gatekeeper-audit; failed to find unique target for patch ~G_~V_Deployment|gatekeeper-audit
```

Reply:
Which version of `kustomize` do you have? It works here..
```
$ git describe
v3.3.0-beta.2-15-g2e14e5a3
$ kustomize version
{Version:kustomize/v3.8.8 GitCommit:72262c5e7135045ed51b01e417a7e72f558a22b0 BuildDate:2020-12-10T18:05:35Z GoOs:linux GoArch:amd64}
$ kustomize build cmd/build/helmify > /dev/null; echo $?
0
```

Reply:
I am seeing it in kustomize v3.8.0 but looks like it's fixed in v3.8.8

Reply:
Since this is fixed in v3.8.8, let's update the Makefile to use v3.8.8 instead